### PR TITLE
added css for select drop down list to match website theme

### PR DESCRIPTION
**Pull request by jefftwin**

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -1,135 +1,150 @@
...
+  --accent-color: #2874a6;
 }
 
 body {
-    padding: 2%;
-    margin: 2%;
-    font-size: 14px;
-    font-family: 'Helvetica', sans-serif;
-    background-color: var(--bg-color);
+  padding: 2%;
```
Comment (maknoon) [inline static/style.css]: can u change ur IDE settings to 4 spaces = 1 tab instead of 2 hahaah

**Review by maknoon — APPROVED** [review]
looks good

File: static/style.css (hunk 1 above)
Comment (jefftwin) [reply static/style.css]: o wot, i thought it was already 4 spaces ... lemme confirm

Comment (jefftwin) [reply static/style.css]: o lel.. it was 2 .. just changed it to 4

**Pull request merged**